keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -0,0 +1,248 @@
+---
+title: go modules
+authors:
+  - "@liggitt"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-api-machinery
+  - sig-release
+  - sig-testing
+reviewers:
```

Comment:
I'll volunteer as a reviewer. Won't have bandwidth until next month, but will lend what I can to help this move forward.

Reply:
thanks, added